tests/components/eafm/test_sensor.py

```diff
@@ -250,7 +251,7 @@ async def test_reading_is_sampled(hass, mock_get_station):
 
     state = hass.states.get("sensor.my_station_water_level_stage")
     assert state.state == "5"
-    assert state.attributes["unit_of_measurement"] == "m"
+    assert state.attributes[ATTR_UNIT_OF_MEASUREMENT] == "m"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)